polaris/polaris/management/commands/process_pending_deposits.py

```diff
@@ -35,151 +34,130 @@
-            PendingDeposits.get_ready_multisig_deposits
-        )()
-        pending_trust_transactions = await sync_to_async(
-            PendingDeposits.get_pending_trust_transactions
-        )()
-        locks = {
-            "source_accounts": defaultdict(asyncio.Lock),
-            "destination_accounts": defaultdict(asyncio.Lock),
-        }
-        async with ServerAsync(settings.HORIZON_URI, client=AiohttpClient()) as server:
-            results = await asyncio.gather(
-                *[
-                    PendingDeposits.process_deposit(t, server, locks)
-                    for t in ready_transactions
-                ],
-                *[
-                    PendingDeposits.handle_submit(t, server, locks)
-                    for t in ready_multisig_transactions
-                ],
-                *[
-                    PendingDeposits.check_trustline(t, server, locks)
-                    for t in pending_trust_transactions
-                ],
-                return_exceptions=True,
-            )
-        await cls.handle_unexpected_exceptions(
-            results,
-            ready_transactions
-            + ready_multisig_transactions
-            + pending_trust_transactions,
-        )
+        Periodically poll for deposit transactions that are ready to be processed
+        and submit them to the CHECK_ACC_QUEUE for verification by the check_accounts_task.
+        """
+        logger.info("check_rails_task started...")
+        while True:
+            await cls.check_rails_for_ready_transactions(qa)
+            await asyncio.sleep(interval)
 
     @classmethod
-    async def handle_unexpected_exceptions(cls, results, processed_transactions):
-        for i, transaction in enumerate(processed_transactions):
-            if results[i] is None:  # no exeption raised
-                continue
-            try:
-                raise results[i]
-            except type(results[i]):
-                logger.exception(
-                    f"An unexpected exception occured while processing transaction {transaction.id}"
-                )
-                transaction.status_message = str(results[i])
-                transaction.status = Transaction.STATUS.error
-                transaction.pending_execution_attempt = False
-                await sync_to_async(transaction.save)()
-                await maybe_make_callback_async(transaction)
-
+    async def check_rails_for_ready_transactions(cls, qa: PolarisQueueAdapter):
+        ready_transactions = await sync_to_async(cls.get_ready_deposits)()
+        for transaction in ready_transactions:
+            transaction.queue = CHECK_ACC_QUEUE
+            transaction.queued_at = datetime.datetime.now(datetime.timezone.utc)
+            transaction.status = Transaction.STATUS.pending_anchor
+            await sync_to_async(transaction.save)()
+            qa.queue_transaction("check_rails_task", CHECK_ACC_QUEUE, transaction)
+        return ready_transactions
```

Comment:
We don't need to return this list right? Looks like `check_rails_task()` doesn't use the return value.